unit_tests/memory_monitor/test_memory_monitor_main.cpp

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#define CATCH_CONFIG_MAIN
+#include "catch2/catch.hpp"
+
+#include <cstdlib>
+#include <malloc.h>
+#include <thread>
+
+#include "memory_monitor_sr.hpp"
+
+// For multithread test
+static const int num_threads = 100;
```

Comment:
```suggestion
constexpr int num_threads = 100;
```
If there is no duplicated definition for `num_threads`, I think we can omit `static` keyword